packages/appeals-service-api/src/lib/notify.js

```diff
@@ -185,12 +170,56 @@ async function sendAppealSubmissionReceivedNotificationEmailToLpa(appeal) {
   }
 }
 
+async function sendStartEmailToLPA(appeal) {
+  if (!isValidAppealForSendStartEmailToLPAEmail(appeal)) {
+    throw new Error('Appeal was not available.');
+  }
+
+  let lpa = {};
+  try {
+    lpa = await getLpa(appeal.lpaCode);
+  } catch (e) {
+    logger.error({ err: e, lpaCode: appeal.lpaCode }, 'Unable to find LPA from given lpaCode');
+  }
+
+  if (!lpa || !lpa.name) {
+    lpa = {
+      name: appeal.lpaCode,
+    };
+  }
+
+  try {
+    if (!lpa.email) {
+      throw new Error('Missing LPA email. This indicates an issue with the look up data.');
+    }
+
+    await NotifyBuilder.reset()
```

Comment:
I think this should be separated into it's own function for function readability.

Reply:
this is its own function. Not sure I follow?